docs/csharp/misc/cs0612.md

```diff
@@ -7,10 +7,11 @@ helpviewer_keywords:
   - "CS0612"
 ms.assetid: 7695f3b7-ffef-43f7-83db-fc1a9e361f1a
 ---
-# Compiler Warning (level 1) CS0612
+# Compiler warning (level 1) CS0612
+
 'member' is obsolete  
   
- The class designer marked a member with the [Obsolete](../programming-guide/concepts/attributes/common-attributes.md#Obsolete) attribute. This means that the member might not be supported in a future version of the class.  
+ The class designer marked a member with the <xref:System.ObsoleteAttribute> attribute. This means that the member might not be supported in a future version of the class.  
```

Comment:
This is now targeting a different article, but I think that's fine.

Reply:
I couldn't find a mention of Obsolete in common-attributes.md. Let me know if I just missed it.

Reply:
It's here: https://github.com/dotnet/docs/blob/master/docs/csharp/language-reference/attributes/general.md#obsolete-attribute